lib/elixir/lib/module.ex

```diff
@@ -861,6 +861,15 @@ defmodule Module do
       end
 
   """
+  def put_attribute(module, key, value)
+
+  def put_attribute(module, key, value) when key in [:moduledoc, :typedoc, :doc] do
+    unless match?({line, _} when is_integer(line), value) do
+      raise ArgumentError, "expected #{inspect(key)} attribute given in the {line, doc} format, got: #{inspect(value)}"
```

Comment:
Should we raise in preprocess attribute instead? Like here: https://github.com/lexmag/elixir/blob/master/lib/elixir/lib/module.ex#L1120?